app/src/test/kotlin/com/mapzen/erasermap/presenter/MainPresenterTest.kt

```diff
@@ -536,6 +607,48 @@ class MainPresenterTest {
         assertThat(mainController.routeRequestCanceled).isTrue()
     }
 
+    @Test fun onBackPressed_shouldShowActionBarInStateRoutePreview() {
+        vsm.viewState = ROUTE_PREVIEW
+        mainController.isActionBarHidden = false
```

Comment:
Should set `mainController.isActionBarHidden = true` here to ensure flag is toggled correctly when back is pressed.